tools/autograd/derivatives.yaml

```diff
@@ -733,6 +736,9 @@
 - name: real(Tensor self) -> Tensor
   self: at::real(grad)
 
+- name: copy_real(Tensor self) -> Tensor
+  self: grad.copy_real()
```

Comment:
@anjali411 shouldn't the result here be a complex number?

Reply:
`copy_real()` returns a complex tensor